File: SDKs/ptv.vic.gov.au/v3/java/lib/src/main/java/openapisdk/models/shared/V3StopTicket.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

public class V3StopTicket {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("is_free_fare_zone")
    public Boolean isFreeFareZone;
    public V3StopTicket withIsFreeFareZone(Boolean isFreeFareZone) {
        this.isFreeFareZone = isFreeFareZone;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("ticket_checks")
    public Boolean ticketChecks;
    public V3StopTicket withTicketChecks(Boolean ticketChecks) {
        this.ticketChecks = ticketChecks;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("ticket_machine")
    public Boolean ticketMachine;
    public V3StopTicket withTicketMachine(Boolean ticketMachine) {
        this.ticketMachine = ticketMachine;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("ticket_type")
    public String ticketType;
    public V3StopTicket withTicketType(String ticketType) {
        this.ticketType = ticketType;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("ticket_zones")
    public Integer[] ticketZones;
    public V3StopTicket withTicketZones(Integer[] ticketZones) {
        this.ticketZones = ticketZones;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("vline_reservation")
    public Boolean vlineReservation;
    public V3StopTicket withVlineReservation(Boolean vlineReservation) {
        this.vlineReservation = vlineReservation;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("zone")
    public String zone;
    public V3StopTicket withZone(String zone) {
        this.zone = zone;
        return this;
    }
}

File: SDKs/ptv.vic.gov.au/v3/python/src/sdk/models/shared/v3_stopticket.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class V3StopTicket:
    is_free_fare_zone: Optional[bool] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('is_free_fare_zone') }})
    ticket_checks: Optional[bool] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ticket_checks') }})
    ticket_machine: Optional[bool] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ticket_machine') }})
    ticket_type: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ticket_type') }})
    ticket_zones: Optional[list[int]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ticket_zones') }})
    vline_reservation: Optional[bool] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('vline_reservation') }})
    zone: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('zone') }})
    
